sklearn/kernel_ridge/krc.py

```diff
@@ -0,0 +1,144 @@
+# Author: Carlos Perales <sir.perales@gmail.com>
```

Comment:
Fwiw, splitting this into separate files provides no advantage imo, but means you often need to edit two or three files at once to make a docstring change. I know we've done such splitting elsewhere in the library, and inconsistently so,  but often not when there is so much shared by the implementations

Reply:
Thanks for the info. Using that Python does not force you to put every object in a separated file, I could move the three objects (base and the inherits ones) into an unique file. Do you prefer so? Or would it break some conventions?

However, as it is now, I think it does not provide so much problems.

Reply:
Personally I'd appreciate them all staying in kernel_ridge.py. That seems much easier to maintain since so much of the code is shared.

Reply:
Done ( 39fdc7aa35169112b4f16c8f9034283abc02e55f ). What's more, I think then no longer a specific folder for `kernel_ridge` is needed.

Reply:
Yes, I had been trying to suggest that a folder was unnecessary

Reply:
Perfect then, sorry for not understanding you before